ansible/roles/test/files/ptftests/fib_test.py

```diff
@@ -38,7 +38,8 @@
 from ptf.testutils import verify_packet_any_port
 from ptf.testutils import verify_no_packet_any
 
-from collections import Iterable, defaultdict
+from collections.abc import Iterable
```

Comment:
Wrap this import by `try...except` to keep backward compatibility?
If this test is backported to old branches using PTF with old python version, then it will fail on old branches.